scripts/diagnose.sh

```diff
@@ -60,7 +60,8 @@ commands=(
 	'$ENG system df'
 	'$ENG volume ls'
 	'systemctl status $ENG --no-pager'
-	'journalctl -n 200 --no-pager -a -u $ENG'
+	'journalctl --no-pager --no-hostname -n 200 -a -u $ENG'
+	'journalctl --no-pager --no-hostname -at balenad'
```

Comment:
Should this have a `-n` argument for consistency with the rest of the PR?

Reply:
I'd rather omit the hard cap on lines, since engine stacktraces can be large on occasion (and this'll specifically filter out all non-engine logs). Perhaps it does make sense to have some really large upper cap just in case, to limit the bandwidth usage of shipping these logs back.